src/cycles/remove-cycles.ts

```diff
@@ -0,0 +1,85 @@
+import * as _ from 'lodash';
```

Comment:
Again, no `lodash` please. You might have to add `lodash.clonedeep` as a dependency